taken from upstream to fix FTBFS on xserver 1.15 and up
http://cgit.freedesktop.org/xorg/driver/xf86-video-geode/commit/?id=9707a7a5eac25536a5150fc12d9ace52420b4d36
http://cgit.freedesktop.org/xorg/driver/xf86-video-geode/commit/?id=70b131d09b7d21df1d1ffcf6a4e50d3086924228

Signed-off-by: Bernd Kuhls <berndkuhls@hotmail.com>

diff -uNr xf86-video-geode-2.11.15.org/src/geode_dcon.c xf86-video-geode-2.11.15/src/geode_dcon.c
--- xf86-video-geode-2.11.15.org/src/geode_dcon.c	2013-10-09 13:47:09.000000000 +0200
+++ xf86-video-geode-2.11.15/src/geode_dcon.c	2014-03-29 12:33:46.718311451 +0100
@@ -29,6 +29,7 @@
 
 /* Includes that are used by all drivers */
 #include <xf86.h>
+#include <xf86Modes.h>
 #include <xf86_OSproc.h>
 #include <compiler.h>
 
